<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthRegForm</class>
 <widget class="QWidget" name="AuthRegForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>455</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0)</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>499</width>
     <height>232</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QLineEdit" name="linePass">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> border-style: outset;
 border-width: 2px;
 border-color: rgb(47, 161, 190)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Hack&quot;;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEmail">
      <property name="styleSheet">
       <string notr="true"> border-style: outset;
 border-width: 2px;
 border-color: rgb(47, 161, 190)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineLogin">
      <property name="styleSheet">
       <string notr="true"> border-style: outset;
 border-width: 2px;
 border-color: rgb(47, 161, 190)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QLabel" name="labelLogin">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 18pt &quot;Hack&quot;;
font: 16pt &quot;Hack&quot;;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelEmail">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Hack&quot;;</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>labelLogin</zorder>
   <zorder>linePass</zorder>
   <zorder>lineEmail</zorder>
   <zorder>lineLogin</zorder>
   <zorder>label</zorder>
   <zorder>labelEmail</zorder>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>501</width>
     <height>113</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonAuth">
      <property name="styleSheet">
       <string notr="true">background-color :rgb(187, 38, 73);
font: 16pt &quot;Hack&quot;;</string>
      </property>
      <property name="text">
       <string>Auth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonReg">
      <property name="styleSheet">
       <string notr="true">background-color :rgb(192, 97, 203)</string>
      </property>
      <property name="text">
       <string>Reg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonChange">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(47, 161, 190)</string>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[ Client ]</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>311</width>
     <height>351</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Hack'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        /| _ ╱|、  &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;       ( •̀ㅅ •́  )&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;      ＿ノ ヽ ノ＼＿ &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    /　`/ ⌒Ｙ⌒ Ｙ　 \&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;   ( 　(三ヽ人　 /　 　|&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;   |　ﾉ⌒＼ ￣￣ヽ　 ノ&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;   ヽ＿＿＿＞､＿＿／&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;       ｜( 王 ﾉ〈 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;       /ﾐ`ー―彡 \ &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;      |╰       ╯|   &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;      |   /\    |&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;      |  /  \   |                    &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
